airflow/providers/google/cloud/operators/functions.py

```diff
@@ -459,4 +459,8 @@ def execute(self, context: 'Context'):
         )
         self.log.info('Function called successfully. Execution id %s', result.get('executionId'))
         self.xcom_push(context=context, key='execution_id', value=result.get('executionId'))
+
+        # HTTP function returns a JSON response, making it available in xcom
+        if 'result' in result:
+            self.xcom_push(context=context, key='function_response', value=result.get('result'))
         return result
```

Comment:
Isn’t the value already available here with an extra `["result"]` call? Making it available again under a different key seems to provide minimal benefit.

Reply:
Same doubt as mine. Closing it - we can reopen if there are some good reasons.